### DISCUSSION TITLE
Password to protect modifications

### DISCUSSION THREAD

**Discussion by Fripix** [linuxserver/Heimdall#1408] [General]
Hello,
I would like to set up Heimdall in a school so that the dashboard is displayed by default in the browsers. However, I would like to password-protect the modification of Heimdall to prevent anyone from making changes without the admin password.
Is this possible?

**Comment by KodeStar** [comment]
Yes? Just set a password?

**Reply by Fripix** [reply]
Yes, set a password to modify the homepage, but not to access it. The goal is that everyone can access Heimdall, but only those with the password can make changes.

**Reply by KodeStar** [reply]
Sorry, I wasn't clear. If you set a password for the admin user and allow public access then people can view the links but not edit them

**Reply by Fripix** [reply]
Oops, I hadn’t seen or understood (how is that possible?).
This is exactly what I want. Thank you!
